src/stash/build.rs

```diff
@@ -101,8 +101,11 @@ fn main() -> anyhow::Result<()> {
     let hashes = fs::File::open(PROTO_HASHES).context("opening proto hashes")?;
     let reader = BufReader::new(&hashes);
     let hashes: Vec<ProtoHash> = serde_json::from_reader(reader)?;
-    let mut persisted: BTreeMap<String, String> =
-        hashes.into_iter().map(|e| (e.name, e.md5)).collect();
+    let mut persisted: BTreeMap<String, String> = hashes
```

Comment:
I don't see how this changes anything. Is the actual fix changing the comparison below?

Reply:
yep!

Reply:
I was comparing the wrong object to the sorted object